### PR TITLE
Openai

### PR DESCRIPTION
**Pull request by Schala**
New:
- python-jsonpatch: lip2wav dep
- python-webrtcvad: lip2wav dep

Updated:
- python-aiohttp 3.7.3
- python-scikit-image 0.18.1
- python-toposort 1.6
- python-tqdm 4.55
- python-zstandard 0.15

### CODE REVIEW

File: mingw-w64-python-aiohttp/PKGBUILD [hunk 1]
```diff
@@ -1,30 +1,46 @@
...
+makedepends=("${MINGW_PACKAGE_PREFIX}-python-setuptools" "${MINGW_PACKAGE_PREFIX}-cython" "git")
+checkdepends=("${MINGW_PACKAGE_PREFIX}-python-freezegun"
+"${MINGW_PACKAGE_PREFIX}-python-pytest-runner"
+"${MINGW_PACKAGE_PREFIX}-python-pytest-cov"
+"${MINGW_PACKAGE_PREFIX}-python-pytest-forked"
+"${MINGW_PACKAGE_PREFIX}-python-pytest-mock"
+"${MINGW_PACKAGE_PREFIX}-python-pytest-timeout"
+"${MINGW_PACKAGE_PREFIX}-python-pytest-xdist"
+"${MINGW_PACKAGE_PREFIX}-python-trustme")
+source=("git+https://github.com/aio-libs/aiohttp#commit=${_gitcommit}"
```
Comment (lazka) [inline mingw-w64-python-aiohttp/PKGBUILD:26]: Why the switch to git?

Comment (Schala) [reply mingw-w64-python-aiohttp/PKGBUILD:26]: It wouldn't work with the source tarballs. I looked at how Arch did it and copied their way and it worked

**Comment by Schala** [comment]
Darn it... it's goofing up again

File: mingw-w64-python-aiohttp/PKGBUILD (hunk 1 above)
Comment (lazka) [reply mingw-w64-python-aiohttp/PKGBUILD:26]: Arch generally prefers git, so they don't have a good reason most of the time.. but ok I guess..

**Pull request merged**